server/views/pages/overview/partials/supportNeedsTabContents.njk

```diff
@@ -2,7 +2,7 @@
   <div class="govuk-grid-column-full">
 
     {% if supportNeeds.problemRetrievingData === true %}
-      There was a problem retrieving the data from Curious.
+      <h2 class="govuk-heading-m">Sorry, the Curious service is currently unavailable.</h2>
```

Comment:
Minor (and we can do it another time), but I wonder if a neater & easier to maintain approach to these views that contain error messages is like what I did in `educationAndTrainingTabContents.njk`

I put the if block right at the top, and I made the true branch handle the happy path (ie. render the data in the various components etc)
I put the error state (sad path) right down the bottom in the else block

The other thing I did was make the if/else/endif statements the outer extent / scope of the template (rather than putting them within the divs).
The reason I did this is because it means the true and false blocks are self contained in respect of their markup .... I'm basically thinking that potentially the error markup might get pulled out into a partial or similar at some stage, so you'd kinda want it "complete" if that makes sense?

Reply:
I think that's a good idea and I've done the same 👍